src/views/SystemIntake/ViewOnly/index.tsx

```diff
@@ -7,11 +8,66 @@ import { SystemIntakeForm } from 'types/systemIntake';
 type SystemIntakeViewOnlyProps = {
   systemIntake: SystemIntakeForm;
 };
-const SystemIntakeView = ({ systemIntake }: SystemIntakeViewOnlyProps) => (
-  <>
-    <h1>Review your Intake Request</h1>
-    <SystemIntakeReview systemIntake={systemIntake} now={DateTime.local()} />
-  </>
-);
+
+function downloadFile(filename: string, content: string) {
+  axios
+    .request({
+      url: `${process.env.REACT_APP_API_ADDRESS}/pdf/generate`,
+      responseType: 'blob',
+      method: 'POST',
+      data: {
+        html: content,
+        filename
+      }
+    })
+    .then(response => {
+      // This approach works fine in the tests I've done in Chrome
+      // With PDF files that are < 100kB. For larger files we might need to
+      // instead redirect the browser to a URL that returns the file. That
+      // approach is complicated by using JWTs for auth.
+      const blobUrl = URL.createObjectURL(
+        new Blob([response.data], { type: 'application/pdf' })
+      );
+
+      const link = document.createElement('a');
+
+      link.href = blobUrl;
+
+      // TODO generate a useful filename here
+      link.download = 'generated.pdf';
+
+      // This downloads the file
+      // TODO test this in various browsers. Some reports say this might not work
+      // properly in Firefox and that firing a MouseEvent is required instead.
+      document.body.appendChild(link);
+      link.click();
+      document.body.removeChild(link);
+    })
+    .catch(error => {
+      // TODO add error handling: display a modal if things fail?
+      console.debug(error);
+    });
+}
+
+const SystemIntakeView = ({ systemIntake }: SystemIntakeViewOnlyProps) => {
+  const download = (): void => {
+    downloadFile('intake.html', document.documentElement.outerHTML);
```

Comment:
How many levels up does `outerHTML` take us? I see in the CSS that you're hiding the header, footer, etc. Any chance we can just avoid them altogether?

Reply:
I ran into some trouble with only grabbing the part of the page that I wanted because some of our styles needed to be nested inside the outer divs to display probably. We can definitely experiment with how to best handle this, though.

One side benefit of the print styles hiding the header and footer is that it makes any page in the app more useful if it's ever printed out.

Reply:
`outerHTML` is the whole enchilada, although in the latest code I'm using `document.body.outerHTML` to only get what is inside the `<body>` tag.